### PR TITLE
ADD: code doc blocks.

### PR DESCRIPTION
**Pull request by yamenarahman**
Just trying to make it beautiful.

### CODE REVIEW

**Comment by yamenarahman** [comment]
@incredimike Always keep your code documented :)

**Comment by yamenarahman** [comment]
@vtalbot This package is meant to be integrated in Laravel. If you navigate Laravel's core you definitely will be impressed how expressive and self explaining it is, but also you will notice that it's well documented and full of code comments. As I said I believe that good code has to be expressive and documented.

File: examples/CountryExample.php [hunk 1]
```diff
@@ -9,6 +9,11 @@ class CountryExample extends Model
 {
 	use \Sushi\Sushi;
 
+    /**
+     * Countries rows.
+     *
+     * @var array $rows
```
Comment (lucasmichot) [inline examples/CountryExample.php]: Laravel docblocks style does not display variable names - this should be `@var array`

File: src/Sushi.php [hunk 2]
```diff
@@ -5,15 +5,34 @@
...
 
+/**
+ * Trait Sushi
+ *
+ * @package Sushi
+ */
 trait Sushi
 {
+    /**
+     * @var $sushiConnection
```
Comment (lucasmichot) [inline src/Sushi.php]: What is the type of the variable, the name is useless

File: src/Sushi.php [hunk 3]
```diff
@@ -5,15 +5,34 @@
...
+     */
     protected static $sushiConnection;
 
+    /**
+     * Resolve connection.
+     *
+     * @param null $connection
```
Comment (lucasmichot) [inline src/Sushi.php]: Not `null`

File: src/Sushi.php [hunk 4]
```diff
@@ -5,15 +5,34 @@
...
+     */
     public static function resolveConnection($connection = null)
     {
         return static::$sushiConnection;
     }
 
+    /**
+     * Boot.
+     *
+     * @throws \ReflectionException
```
Comment (lucasmichot) [inline src/Sushi.php]: This method does not throw any exception, and the `@return void`

File: src/Sushi.php [hunk 5]
```diff
@@ -57,6 +76,11 @@ public static function bootSushi()
         }
     }
 
+    /**
+     * Set Sqlite connection.
+     *
+     * @param $database
```
Comment (lucasmichot) [inline src/Sushi.php]: Missing type

Comment (lucasmichot) [reply src/Sushi.php]: Missing `@return void`

File: src/Sushi.php [hunk 6]
```diff
@@ -65,6 +89,9 @@ protected static function setSqliteConnection($database)
         ]);
     }
 
+    /**
+     * Migrate.
+     */
```
Comment (lucasmichot) [inline src/Sushi.php]: Missing `@return void`

**Comment by lucasmichot** [comment]
@yamenarahman , adding docblocs is a great initiative. But:
- they should comply Laravel style
- the should actually reflect the real type
- They should be meaningful

I guess this PR needs some changes and additions

**Comment by yamenarahman** [comment]
@lucasmichot Thanks alot for your comments.

**Comment by calebporzio** [comment]
Thanks for the contribution, however, I personally don't use DocBlocks, I know there is a pretty strong case for adding them to a package that will be consumed by others, and in that case here is what I would consider:

* No "comments" in the docblocks, especially if they don't offer any useful information not present in the code.
* Only DocBlocks on methods that will be extended or consumed from the user (which is basically none of them right now)

Let's hold off on DocBlocks for now, thanks for being willing to contribute though!

**Pull request closed without merging**